src/components/Pokemon.jsx

```diff
@@ -0,0 +1,44 @@
+import React, {useEffect, useState} from 'react';
+import axios from "axios";
+import './Pokemon.css';
+
+
+
+const Pokemon = ({endpoint}) => {
+
+
+    useEffect(() => {
+        async function findpokemon(){
+            try{ const response = await axios.get(endpoint)
+                setPokemon(response.data)
+            }catch(e){
+                console.error(e)
+            }
+        }
+        void findpokemon()
+    }, [endpoint]);
+
+    const [pokemon,setPokemon] = useState("")
+
+
+    return (
+        <article className="poke-card">
+            {Object.keys(pokemon).length > 0 &&
```

Comment:
Als de default waarde van `pokemon` hier óók `undefined` zou zijn, dan kan deze check een stuk makkelijker
```suggestion
            {pokemon &&
```